plugins/modules/ovirt_disk.py

```diff
@@ -374,35 +393,13 @@ def transfer(connection, module, direction, transfer_func):
         elif auth.get('ca_file'):
             context.load_verify_locations(cafile=auth.get('ca_file'))
 
-        try:
-            destination_url = urlparse(transfer.transfer_url)
-            transfer_connection = HTTPSConnection(
-                destination_url.hostname,
-                destination_url.port,
-                context=context,
-            )
-            transfer_func(
-                transfer_service,
-                transfer_connection,
-                destination_url,
-                transfer.signed_ticket
-            )
-        except Exception as e:
-            module.warn("Image transfer fail with message: {}.".format(e))
-            module.warn("Retrying trasfer with proxy.")
-            destination_url = urlparse(transfer.proxy_url)
-            transfer_connection = HTTPSConnection(
-                destination_url.hostname,
-                destination_url.port,
-                context=context,
-            )
-
-            transfer_func(
-                transfer_service,
-                transfer_connection,
-                destination_url,
-                transfer.signed_ticket
-            )
+        transfer_connection, url = create_transfer_connection(module, transfer, context)
```

Comment:
In this context, using "connection" instead of transfer_connection would be little nicer. You agree with me since
you used "url" and not "transfer_url" :-)

You can also rename url to dest_url to be more consistent with other names.

Reply:
cannot use `connection` it is already defined as another variable 
so if we want to be consistent maybe use the `transfer_url`